mediawiki.conf

```diff
@@ -1,6 +1,8 @@
 # VisualEditor support. T262392
 AllowEncodedSlashes NoDecode
 RewriteRule ^/?w/rest.php/ - [L,NC]
+# Image authorization support
+RewriteRule ^/?w/img_auth.php/ - [L,NC]
```

Comment:
could you please comment on the reason to have this rule both on apache config and `.htaccess`?

Reply:
I can't remember now, probably we can remove the rewrite rules from mediawiki.conf file.
they don't work because rewrite rules in .htaccess file have higher priority.
Rewrite rules in mediawiki.conf file may be used in a wiki farm, but as far as I remember I created symlink to .htaccess too, so I think we can remove them in mediawiki.conf but we have to make sure that it works fine, probably we need to create a separate task for this

Reply:
ok, if right now it works as is, let's leave it for now